JuliaExperimental/julia/gaprat.jl

```diff
@@ -319,40 +319,40 @@ end
 ##
 
 function numerator( a::GAPRat )
-    ptr = ccall( Main.gap_NUM_RAT, Ptr{Cvoid},
+    ptr = ccall( :NUM_RAT, Ptr{Cvoid},
                 (Ptr{Cvoid},), a.obj.ptr )
     return GAPRat( GapObj(ptr) )
 end
 
 function denominator( a::GAPRat )
-    ptr = ccall( Main.gap_DEN_RAT, Ptr{Cvoid},
+    ptr = ccall( :DEN_RAT, Ptr{Cvoid},
                 (Ptr{Cvoid},), a.obj.ptr )
     return GAPRat( GapObj(ptr) )
 end
 
 #T This works only for GAP integers, GAP has no C function for GAP rationals?
 function abs( a::GAPRat )
-    ptr = ccall( Main.gap_AbsInt, Ptr{Cvoid},
+    ptr = ccall( :AbsInt, Ptr{Cvoid},
                 (Ptr{Cvoid},), a.obj.ptr )
     return GAPRat( GapObj(ptr) )
 end
 
 function gcd( a::GAPRat, b::GAPRat )
-    ptr = ccall( Main.gap_GcdInt, Ptr{Cvoid},
+    ptr = ccall( :GcdInt, Ptr{Cvoid},
```

Comment:
Meta-level: note that I don't believe this code is correct to start with, see issue #71. Alas, of course you are just converting it here. Once we start adding tests for it, we should quickly see if I am right or not :)

Reply:
To be fair, this is just a conversion. This code is broken (for example the GapObj call can be removed, and so on. Once this is merged, I will clean up this whole file again

@ThomasBreuer be prepared ;)

Reply:
Fair enough